kube/pod.go

```diff
@@ -204,9 +204,19 @@ func getEnvVars(role *model.Role, defaults map[string]string, secrets SecretRefM
 			continue
 		}
 
-		ok, stringifiedValue := configValue(config, defaults)
-		if !ok {
-			continue
+		var stringifiedValue string
+		if createHelmChart {
+			required := ""
+			if config.Required {
+				required = fmt.Sprintf(`required "A valid .Values.env.%s is required" `, config.Name)
```

Comment:
Alright. That message is sort of hostile to the people who actually would be running helm (who wouldn't be writing the charts, in our case), but it has _enough_ information that it's still parsable.